Source/AwsCommonRuntimeKit/crt/ByteBuffer.swift

```diff
@@ -271,15 +270,14 @@ extension ByteBuffer: AwsStream {
 extension ByteBuffer {
     public convenience init(data: Data) {
         self.init(size: data.count)
-        for byte in data {
-            array.append(byte.byteSwapped)
-        }
+        let byteArray: [UInt8] = data.map { $0 }
```

Comment:
Is there a byte swap still happening here magically? should this be `$0.byteSwapped`?

Reply:
idk, I stole this code from someone online. let me test. idk how this magic is occurring really, it was purely theft on my part.

Reply:
turns out you didnt need the byte swapped to get the right type, I just thought you did. the magic works and should be the default.